grails-core/src/main/groovy/org/grails/core/exceptions/DefaultErrorsPrinter.groovy

```diff
@@ -47,7 +47,7 @@ class DefaultErrorsPrinter extends DefaultStackTracePrinter implements CodeSnipp
         res == null ? te.className : res.getFilename()
     }
 
-    String prettyPrintCodeSnippet(Throwable exception) {
+    String prettyPrintCodeSnippet(Throwable exception, Map attrs = [:]) {
```

Comment:
Using Groovy default params on overridden/implemented methods makes the code hard to follow. Can we use:
```groovy
@Override
String prettyPrintCodeSnippet(Throwable t) {
    prettyPrintCodeSnippet(t, [:])
}

String prettyPrintCodeSnippet(Throwable t, Map attrs) {
    
}
```

(or change the interface to include both versions)